### PR TITLE
Students answer questions

### PR DESCRIPTION
**Pull request by naderakhgari**
Thank you for contributing! To help the review process, please provide the following:

### Proposal
Students can select a quiz and answer the questions of that quiz

### Related
Provide links to any associated pull requests or issues. You can use `#123` to link to a PR or issue in this repository, or `user/repo#123` to link to other repositories.

### Checklist

- [ ] I have read the [code of conduct] and [contributing guide]
- [ ] I have made this pull request to the `master` branch
- [ ] I have run all of the automated validation using `npm run ship`
- [ ] I have added myself to the `"contributors"` list in the `package.json` (or do not want to)

[code of conduct]: https://github.com/textbook/starter-kit/blob/master/.github/CODE_OF_CONDUCT.md
[contributing guide]: https://github.com/textbook/starter-kit/blob/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -14,12 +14,14 @@ import NewQuestion from "./Components/NewQuestion.js";
...
 import Questions from "../src/mockData/Questions.json";
 export function App() {
-
-	// const [message, setMessage] = useState("Loading...");
-
-	// useEffect(() => {
-	// 	getMessage().then((message) => setMessage(message));
-	// }, []);
+	const [questionData, setQuestionData]=useState({});
+	const [route, setRoute]=useState("quizes");
```
Comment (pendletons) [inline client/src/App.js:18]:
```suggestion
	const [route, setRoute]=useState("quizzes");
```

File: client/src/Components/Questions.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
+import React, { useState, useEffect } from "react";
+import { Button, Form, FormGroup, Label, Input } from "reactstrap";
+import dayjs from "dayjs";
+
+const Questions = (props) => {
+
+	const [i, setI] = useState(0);
```
Comment (pendletons) [inline client/src/Components/Questions.js:7]: I would rename this something more descriptive, like `currentQuestionIndex`.

**Review by pendletons — CHANGES_REQUESTED** [review]
👍 Looks good overall, just a couple of comments.

File: client/src/Components/Questions.js (hunk 2 above)
Comment (naderakhgari) [reply client/src/Components/Questions.js:7]: Yes, I agree with you

File: client/src/App.js (hunk 1 above)
Comment (naderakhgari) [reply client/src/App.js:18]: Thanks Shanthi, it was my mistake at the beginning used quizes but I'm going to fix it.

**Review by pendletons — APPROVED** [review]

**Comment by naderakhgari** [comment]
Students can choose a quiz and answer the questions

File: client/src/Components/Questions.js [hunk 3]
```diff
@@ -27,7 +25,11 @@ const Questions = (props) => {
 			.then((res) => res.json())
 			.then((data) => console.log(data))
 			.catch((err) => console.error(err));
-	}, [props.fetchedData.questions_id[i], route, requestOption]);
+	}, [
+		props.fetchedData.questions_id[currentQuestionIndex],
```
Comment (yusuf963) [inline client/src/Components/Questions.js]: good idea to handle an error

**Pull request merged**